server/sandbox_run_linux.go

```diff
@@ -187,6 +196,21 @@ func (s *Server) configureSandboxIDMappings(mode string, sc *types.LinuxSandboxS
 				ret.AutoUserNsOpts.AdditionalGIDMappings = m
 			}
 		}
+		// make sure we haven't asked to map any sensitive and/or privileged IDs
+		if minimumMappableUID >= 0 {
+			for _, uidSlice := range ret.AutoUserNsOpts.AdditionalUIDMappings {
+				if int64(uidSlice.HostID+uidSlice.Size-1) < minimumMappableUID {
```

Comment:
should we check for `uidSlice.HostID < minimumMappableUID` instead of `uidSlice.HostID+uidSlice.Size-1 < minimumMappableUID`?

If HostID is `>= minimumMappableUID` then each other ID in the mapping is also `>= minimumMappableUID`

(same comment for the other checks)

Reply:
Yes, it should.  Changing it.